Sources/App/Models/License.swift

```diff
@@ -151,6 +151,15 @@ enum License: String, Codable, Equatable, CaseIterable {
         case other
         case incompatibleWithAppStore = "incompatible"
         case compatibleWithAppStore = "compatible"
+        
+        var userFacingString: String {
+            switch self {
+            case .none: return License.none.shortName
+            case .other: return License.other.shortName
+            case .incompatibleWithAppStore: return "App Store incompatible"
+            case .compatibleWithAppStore: return "App Store compatible"
```

Comment:
same with "compatible with the App Store"

Reply:
Hadn't actually changed this one 😀 

I don't disagree with the feedback, just a limitation with how I've currently got it setup. Won't be too hard to tweak though so will update this PR in the morning with amendments.

Reply:
Yea I wasn't meaning to make "is" specific for this case. I think it would work as a blanket replacement for "matches"